app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,8 +1,9 @@
-<p>
+<p class="h1">
   <%= t('.greeting',
         recipient: @resource.email,
         locale: @resource.locale,
-        default: "Welcome #{@resource.email}!") %>
+        name: name.present ? name : t('no_name_fallback'),
```

Comment:
This is not going to work, as `name` will bean undefined local variable. It should be
```
name: @resource.name.present? ? @resource.name : t('no_name_fallback'),
```
But I do not understand this anyway. In `devise.de.yml`, `confirmation_instructions.greeting` does not have a `name` variable (it is just "Willkommen bei MaMpf"), the same goes for `devise.en.yml`. So, that is what will be shown. Furthermore, at this point in the registration process, the `name` attribute of the user who registers is still set to `nil`. So, the above ternary operator will always give `t('no_name_fallback')`.

Reply:
It was too late when I worked on this, thanks for spotting my mistake! :hankey: